packages/stark-core/src/modules/session/session.module.ts

```diff
@@ -47,11 +48,11 @@ export class StarkSessionModule {
 
 	/**
 	 * Prevents this module from being re-imported
-	 * @link https://angular.io/guide/singleton-services#prevent-reimport-of-the-coremodule
-	 * @param parentModule - the parent module
-	 * @param routingService - The routing service of the application
-	 * @param sessionConfig - The configuration of the session module
-	 * @param appInitStatus - A class that reflects the state of running {@link APP_INITIALIZER}s
+	 * See {@link https://v7.angular.io/guide/singleton-services#prevent-reimport-of-the-greetingmodule|Angular docs: Prevent reimport of a root module}
```

Comment:
For info, this doc is not available in the documentation. 
https://stark.nbb.be/api-docs/stark-core/latest/modules/StarkSessionModule.html

Reply:
In any case, it doesn't harm to keep it ;)